src/call.c

```diff
@@ -94,6 +95,10 @@ static const char *state_name(enum call_state st)
 
 static void set_state(struct call *call, enum call_state st)
 {
+	call->was_established =
+		st == CALL_STATE_TERMINATED &&
+		call_is_established(call);
```

Comment:
another option is to set this flag in the established_handler

Reply:
@juha-h I understand your arguments. But the combination CALL_STATE_ESTABLISHED and not on hold locally is used very often. Thus it reduces the code.

In order to avoid the confusion with "ESTABLISHED" the suggestion of @alfredh is good. I will rename the functions to:
```
bool call_is_active(const struct call *call);
bool call_was_active(const struct call *call);
```